src/EditorFeatures/VisualBasicTest/CodeActions/InlineTemporary/InlineTemporaryTests.vb

```diff
@@ -4401,5 +4401,28 @@ End Class
             Await TestInRegularAndScriptAsync(code, expected, ignoreTrivia:=False)
         End Function
 
+        <Fact(Skip:="InvalidCastException"), Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
```

Comment:
It is the link below: [https://github.com/dotnet/roslyn/issues/16697](https://github.com/dotnet/roslyn/issues/16697)

---
In reply to: [115317743](https://github.com/dotnet/roslyn/pull/18416#discussion_r115317743) [](ancestors = 115317743)